vpr/src/draw/draw.h

```diff
@@ -32,20 +33,16 @@ void alloc_draw_structs(const t_arch* arch);
 void free_draw_structs();
 
 #ifndef NO_GRAPHICS
-
-void draw_get_rr_pin_coords(int inode, float* xcen, float* ycen, const e_side& pin_side);
-void draw_get_rr_pin_coords(const t_rr_node& node, float* xcen, float* ycen, const e_side& pin_side);
-
-void draw_triangle_along_line(ezgl::renderer* g, ezgl::point2d start, ezgl::point2d end, float relative_position = 1., float arrow_size = DEFAULT_ARROW_SIZE);
-void draw_triangle_along_line(ezgl::renderer* g, ezgl::point2d loc, ezgl::point2d start, ezgl::point2d end, float arrow_size = DEFAULT_ARROW_SIZE);
-void draw_triangle_along_line(ezgl::renderer* g, float xend, float yend, float x1, float x2, float y1, float y2, float arrow_size = DEFAULT_ARROW_SIZE);
-
 const ezgl::color SELECTED_COLOR = ezgl::GREEN;
 const ezgl::color DRIVES_IT_COLOR = ezgl::RED;
 const ezgl::color DRIVEN_BY_IT_COLOR = ezgl::LIGHT_MEDIUM_BLUE;
 
 const float WIRE_DRAWING_WIDTH = 0.5;
 
+t_edge_size find_edge(int prev_inode, int inode);
```

Comment:
Some of these have comments in the .cpp file --> copy or move those "what do I do" comments here.  If you know what some other functions do, go ahead and comment them here too. (Don't worry if you can't comment them all -- some is better than none).